File: src/main/java/um/tesoreria/facturador/model/dto/TipoChequeraDto.java
package um.tesoreria.facturador.model.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class TipoChequeraDto {

    private Integer tipoChequeraId;
    private String nombre = "";
    private String prefijo = "";
    private Integer geograficaId = 1;
    private Integer claseChequeraId = 2;
    private Byte imprimir = 0;
    private Byte contado = 0;
    private Byte multiple = 0;
    private String emailCopia;
    private GeograficaDto geografica;
    private ClaseChequeraDto claseChequera;

}

File: src/main/java/um/tesoreria/facturador/kotlin/model/dto/TipoChequeraDto.kt
package um.tesoreria.facturador.kotlin.model.dto

data class TipoChequeraDto(

    var tipoChequeraId: Int? = null,
    var nombre: String = "",
    var prefijo: String = "",
    var geograficaId: Int = 1,
    var claseChequeraId: Int = 2,
    var imprimir: Byte = 0,
    var contado: Byte = 0,
    var multiple: Byte = 0,
    var geografica: GeograficaDto? = null,
    var claseChequera: ClaseChequeraDto? = null

)
